### PR TITLE
Suite-Desktop: Fit text in bridge modal

### LINKED ISSUE #2950: Update Trezor Bridge modal doesn't fit to Suite window and offers Linux version of Bridge
**Describe the bug**
+ when maximized and in full HD version of Suite modal doesn't fit to Suite window.
+ see screenshot, you have to scroll down to see that you can close this modal and drop down menu doesn't fit suite at all.
+ even i'm in Windows Suite offers me `Linux` version of Bridge.
 
**Info:**
 - Suite Version: app, commit 36c7716bd5c2a01de4b8981a2ac680f80c79164c
 - OS: Microsoft Windows [Version 10.0.19042.630]
 - Firmware Version: 2.3.4

**Steps to reproduce:**
1. install/use Trezor Bridge older than 2.0.30
2. install current public Suite version 2020.11.1
3. click on Trezor Bridge update notification

**Expected result:**
Modal is smaller to fit to suite window

**Actual result:**
Modal doesn't fit to suite window

**Screenshots:**
image

### PR DESCRIPTION
**Pull request by keraf**
Partially fixes #2950 (design part)

### CODE REVIEW

**Review by mroz22 — APPROVED** [review]
OK, you sure you want to close the issue (it has 'fixes' keyword prepended) but also 'partially', so I am not sure whether it was your intention

**Comment by tsusanka** [comment]
Ping @keraf regarding @mroz22's question.

**Comment by keraf** [comment]
I dug a bit more into the second part of #2950 and it's related to connect. I'll open an issue in there and fix it there. For now I guess we can close the original issue when this one gets merged. 
[
trezor/connect#750](https://github.com/trezor/connect/issues/715)

File: packages/suite/src/views/suite/bridge/index.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const BridgeDesktopNote = styled(P)`
 `;
 
 const StyledImage = styled(Image)`
-    @media screen and (max-height: ${variables.SCREEN_SIZE.MD}) {
+    @media screen and (max-height: ${variables.SCREEN_SIZE.LG}) {
```
Comment (slowbackspace) [inline packages/suite/src/views/suite/bridge/index.tsx:82]: No images for the poor! 😁  This should seriously be handled better on the design side. Modals should be designed so they fit to most of screens (768px height is a still good rule of thumb). Assuming everyone has at least full hd display (with suite almost on fullscreen) is wrong, but 🤷‍♂️. (Rant is aimed for a product/design team, not you. I just couldn't help myself).

**Pull request merged**